docs/api-reference/carto/overview.md

```diff
@@ -133,14 +133,15 @@ The CARTO submodule includes the CartoLayer that simplifies the interaction with
     import { getData } from '@deck.gl/carto';
     import { H3HexagonLayer } from '@deck.gl/geo-layers/';
 
-    const data =  await getData({
+    const result = await getData({
       type: MAP_TYPES.QUERY,
       source: `SELECT bqcarto.h3.ST_ASH3(internal_point_geom, 4) as h3, count(*) as count
                   FROM bigquery-public-data.geo_us_census_places.us_national_places 
                 GROUP BY h3`,
       connection: 'connection_name',
       format: 'json'
```

Comment:
I'd update this line to use the corresponding constant from the FORMATS enumeration